src/components/questions/AudioRecorder.tsx

```diff
@@ -131,28 +138,68 @@ const AudioRecorder = (props: { onAudioRecorded: (recordUri: string) => void })
       <View style={styles.container}>
         <View style={styles.rowContainer}>
           {isRecording ? (
-            <Icon name="stop" size={48} style={styles.icon} onPress={onStopRecord} />
+            <Icon
+              name="stop"
+              size={48}
+              accessibilityLabel={'Stop opname knop'}
```

Comment:
"Spraakopname stoppen". Dat het een knop is hoor je aan het geluidje en wordt ook voorgelezen (op Android iig)